src/main/QueryServer.cpp

```diff
@@ -132,7 +133,11 @@ QueryServer::getLedgerEntryRaw(std::string const& params,
 
     if (!keys.empty())
     {
-        auto& bl = *mBucketListSnapshots.at(std::this_thread::get_id());
+        auto snapshotPtr = mBucketListSnapshots.at(std::this_thread::get_id());
```

Comment:
This needs to be `auto& snapshotPtr`, since currently we never actually update the snapshot in the vector and would have to get a new copy on every call.

Reply:
yeah sorry, it was as of last commit, but got removed by accident. Fixed now.